### DISCUSSION TITLE
"CommitFailureException" in linux containers using persistent storage

### DISCUSSION THREAD

**Discussion by winpunk** [microsoft/FASTER#888] [Q&A]
Hi,

We are using Faster with persistent storage in Openshift as an outbox component for Kafka message sending in a high load application and sometimes some pods randomly starts throwing these exceptions:

`FASTER.core.CommitFailureException: Commit of address range [2170087932-2170089232] failed with error code 4294967295    at FASTER.core.FasterLogScanIterator.SlowWaitAsync(FasterLogScanIterator this, CancellationToken token)    at FASTER.core.FasterLogScanIterator.GetAsyncEnumerable(MemoryPool`1 pool, CancellationToken token)+MoveNext()    at FASTER.core.FasterLogScanIterator.GetAsyncEnumerable(MemoryPool`1 pool, CancellationToken token)+System.Threading.Tasks.Sources.IValueTaskSource<System.Boolean>.GetResult() ....`

What does "error code 4294967295" mean? What could be a problem?

**Comment by badrishc** [comment]
It probably means you ran out of disk space there.

**Reply by winpunk** [reply]
We checked that, only 2% of disk space was used at that time.
